internal/serverinstall/nomad.go

```diff
@@ -756,30 +748,22 @@ func (i *NomadInstaller) runJob(
 	}
 
 	s.Update("Waiting for allocation to be scheduled")
-EVAL:
 	qopts := &api.QueryOptions{
```

Comment:
Related to my later comment, we probably need to be setting the WaitTime (or whatever it's called) on this so that we don't block indefinitely on these calls, but rather wait some amount of time before we figure out if we want to continue waiting.

Reply:
The API tells me they use a default `WaitTime`, so we shouldn't be blocking forever. I set a default one anyway to be sure.